README.md

```diff
@@ -1,4 +1,4 @@
-# Weaviate quickstart code - TypeScript
+# Weaviate Quickstart code - TypeScript
```

Comment:
Not sure about this one. I think "quickstart" should be lower case since it's not a product name and we use a mostly lowercase title style.

Reply:
sure, i get that!